Gopkg.toml

```diff
@@ -49,6 +49,17 @@ ignored = ["github.com/openshift/installer/tests*"]
   name = "k8s.io/client-go"
   version = "6.0.0"
 
+[[constraint]]
+  name = "sigs.k8s.io/cluster-api"
+  branch = "master"
```

Comment:
Do we really want these floating with master?  Can we pin to their current tips and then manually bump when they make changes?

Reply:
`master` is fine